src/components/bulkConcepts/component/ConceptTable.jsx

```diff
@@ -24,7 +24,7 @@ const ConceptTable = ({
         currentPage={currentPage}
         loading={loading}
         PaginationComponent={ConceptPagination}
-        defaultPageSize={concepts.length <= conceptLimit ? concepts.length : conceptLimit}
```

Comment:
Do you happen to know why this code was originally written as so?

Reply:
Seen... and shall revert

Reply:
I have made some changes to the code. Please share your thoughts

Reply:
I have made some changes to the code. Please share your thoughts

Reply:
How was it able to work with your original changes?

Reply:
Looking through both the code on the `ConceptTable` page and `BulkConceptsPage`, I realized that the defaultPageSize being occasionally set to  1 and unmounted() made it render with the same value when the component mounts again.

The quick fix was to adjust the defaultPageSize to a fixed value of 10 as the functionality for filtering of concepts when searching is handled by the `searchOption` function in the `BulkConceptsPage` making the ternary on `defaultPageSize` redundant.

Reply:
Why a value of 10?

Reply:
The POs have always been of the view that when rendering the `Add CIEL Concepts` page, the user would see 10 CIEL concepts at a time so the `conceptLimit`. This is further addressed in `OCLOMRS-316` variable was created in November 2018 and equated to 10.

Reply:
But i did not see 10 in your original changes

Reply:
You saw `conceptLimit` in my original changes which is set to 10 in `bulkConceptsPage: line 57` and then imported into `ConceptTable` on `line 10`